tonic-reflection/src/server/mod.rs

```diff
@@ -1,11 +1,307 @@
-pub use crate::pb::v1::server_reflection_server::{ServerReflection, ServerReflectionServer};
-
-use prost::DecodeError;
+use std::collections::HashMap;
 use std::fmt::{Display, Formatter};
+use std::sync::Arc;
+
+use prost::{DecodeError, Message};
+use prost_types::{
+    DescriptorProto, EnumDescriptorProto, FieldDescriptorProto, FileDescriptorProto,
+    FileDescriptorSet,
+};
+use tonic::Status;
 
 mod v1;
+pub use v1::{ServerReflection, ServerReflectionServer}; // For backwards compatibility
+mod v1alpha;
```

Comment:
Should this be located below `mod v1;` line?

Reply:
Personally I prefer to keep the `use v1::` imports close to the `mod v1` declaration.